cpp/skyweaver/src/skyweaver_cli.cu

```diff
@@ -447,7 +447,20 @@ int main(int argc, char** argv)
                  [](std::size_t nthreads) { omp_set_num_threads(nthreads); }),
              "The number of threads to use for incoherent dedispersion")
 
-            // Logging options
+            ("wait-for-space",
+              po::value<std::vector<std::string>>()
```

Comment:
Why is this a vector of strings and not just a string. If you meant for it to be a std::vector<char> (which is what it effectively is) then use that.

Maybe I am not understanding the use case here, but isn't the argument defined like:

```
--wait-for-space a:b:c
```

If this is the case then it is not multitoken.

Reply:
Yes, it is not a multitoken. Treat the argument as a string now.